tests/integration/integration_steps/fabric_steps.py

```diff
@@ -12,10 +14,25 @@ def read_deltalake_timeseries(timeseries_path:str, credential: DefaultAzureCrede
     df = delta_table.to_pandas()
     return df
 
+def prepare_lakehouse_dataframe_for_comparison(dataframe, external_id):
+    dataframe = dataframe.loc[dataframe["externalId"] == external_id]
+    if dataframe['timestamp'].dt.tz is None:
```

Comment:
You use the string 'timestamp' 11 times in 9 lines of code. Should create constants for strings like this to prevent typos or unnecessary memory allocation.